tests/AutoReview/FixerFactoryTest.php

```diff
@@ -83,6 +83,7 @@ public function provideFixersPriorityCases()
             [$fixers['function_to_constant'], $fixers['no_trailing_whitespace']], // tested also in: function_to_constant,no_trailing_whitespace.test
             [$fixers['function_to_constant'], $fixers['no_whitespace_in_blank_line']], // tested also in: function_to_constant,no_whitespace_in_blank_line.test
             [$fixers['indentation_type'], $fixers['phpdoc_indent']],
+            [$fixers['is_null'], $fixers['yoda_style']],
```

Comment:
please add comment with a file as well (as when there is lack of comment, we know we don't have itest for given priority test yet)

Reply:
added and rebased